components/canvas/webgl_thread.rs

```diff
@@ -1179,6 +1181,56 @@ impl WebGLImpl {
         chan.send(location).unwrap();
     }
 
+    #[allow(unsafe_code)]
```

Comment:
Why allow unsafe code here?

Reply:
No reason to. Probably added it accidentally, thanks for noticing!